portal/src/pages/my/privacy.vue

```diff
@@ -125,20 +126,42 @@ export default {
       if (isNil(this.user) || isNil(this.user.permissions)) {
         return []
       }
-      return (
+      let permissions =
         this.user.permissions.filter(
-          permission => permission.type !== 'Cookies'
+          permission => !permission.is_notification_only
         ) || []
-      )
+      if (!this.isAuthenticated) {
+        permissions = permissions.filter(
+          permission => !permission.is_after_login
+        )
+      }
+      return permissions
     },
     cookies() {
       if (this.user && this.user.permissions) {
         return this.user.permissions.find(
           permission => permission.type === 'Cookies'
         )
       } else return false
+    },
+    submitButtonLabel() {
+      if (
+        this.$store.getters.hasGivenCommunityPermission &&
```

Comment:
Kan dit niet worden versimpeld naar `if (this.$store.getters.hasGivenCommunityPermission !== this.hasInitialCommunityPermission)`?

Reply:
Ja, ik had aanvankelijk een 'save-privacy-settings-and-login' vertaling, maar nu die er niet meer is kan het wel simpeler. Zal ik later doorvoeren.